plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/DBNUtils.java

```diff
@@ -48,6 +46,8 @@
 public class DBNUtils {
 
     private static final Log log = Log.getLog(DBNUtils.class);
+    private static final Comparator<DBNNode> ALPHANUMERIC_COMPARATOR_COMPARATOR = (o1, o2) -> AlphanumericComparator.getInstance()
```

Comment:
Maybe just `ALPHANUMERIC_COMPARATOR`?
```suggestion
    private static final Comparator<DBNNode> ALPHANUMERIC_COMPARATOR = (o1, o2) -> AlphanumericComparator.getInstance()
```

Reply:
fixed

Reply:
`Comparator.comparing(DBNNode::getNodeDisplayName, AlphanumericComparator.getInstance())`